docs/docs/configuration/auth.md

```diff
@@ -163,7 +163,7 @@ Whether you are using GitLab.com or self-hosting GitLab, follow [these steps to
 
 If you need projects filtering, add the extra `read_api` scope to your application.
 
-The following config should be set to ensure that the oauth will work properly. To get a cookie secret follow [these steps](https://github.com/oauth2-proxy/oauth2-proxy/blob/master/docs/configuration/configuration.md#configuration)
+The following config should be set to ensure that the oauth will work properly. To get a cookie secret follow [these steps](https://github.com/oauth2-proxy/oauth2-proxy/blob/master/docs/docs/configuration/overview.md)
```

Comment:
Perhaps this link should point to the correct place on the docs site instead of the raw markdown, WDYT?

Reply:
That would be the best. The link to blob was there so I didn't want to change it thinking that there might have been a reason behind it. :D

Reply:
Ok, I think this is the proper way of generating site links.